### PR TITLE
Zh_cn/community translation

### PR DESCRIPTION
**Pull request by liuzxc**
Community translation, including: Conferences, Mailing List, Ruby-Core, User Groups and Blog.

Related #1074.

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho @cursorzz @isjia @debbbbie

Please review. :mag_right: :mag_right: :mag_right:

File: zh_cn/community/ruby-core/index.md [hunk 1]
```diff
@@ -1,7 +1,117 @@
 ---
 layout: page
-title: "Ruby 内核 - Ruby 官方网站"
+title: "Ruby 内核"
```
Comment (aidewoode) [inline zh_cn/community/ruby-core/index.md]: Hi, I think the "Ruby Core" translate to "Ruby 核心" will be better. You know "内核"  means to "Kernel" .

**Comment by liuzxc** [comment]
@aidewoode  I agree with you, "Ruby 核心" is better.

**Comment by stomar** [comment]
Please provide a **new PR**:
* **never merge master into your topic branch**
* squash into a single commit; commit message should be `Translate community pages (zh_cn)`

**Pull request closed without merging**